[Bug]: bs-logger is missing from the dependencies

### Version

11.0.0

### Steps to reproduce

N/A

### Expected behavior

`bs-logger` should be a dependency of jest-preset-angular since it's used in `ng-jest-transformer`

### Actual behavior

```
Cannot find module 'bs-logger'
Require stack:
- C:\www\bit-capsules\62121062bf6973ee4631fe14f96ac4f1a3a927b4\node_modules\.pnpm\registry.npmjs.org+jest-preset-angular@11.0.0_a7ccfb11043122fcd72a8780d7f7f03f\node_modules\jest-preset-angular\build\ng-jest-transformer.js
- C:\www\bit-capsules\62121062bf6973ee4631fe14f96ac4f1a3a927b4\node_modules\.pnpm\registry.npmjs.org+jest-preset-angular@11.0.0_a7ccfb11043122fcd72a8780d7f7f03f\node_modules\jest-preset-angular\build\index.js
- C:\www\bit-angular\node_modules\.pnpm\registry.npmjs.org+jest-util@27.3.1\node_modules\jest-util\build\requireOrImportModule.js
- C:\www\bit-angular\node_modules\.pnpm\registry.npmjs.org+jest-util@27.3.1\node_modules\jest-util\build\index.js
- C:\www\bit-angular\node_modules\.pnpm\registry.npmjs.org+@jest+core@27.3.1_ts-node@9.1.1\node_modules\@jest\core\build\FailedTestsInteractiveMode.js
- C:\www\bit-angular\node_modules\.pnpm\registry.npmjs.org+@jest+core@27.3.1_ts-node@9.1.1\node_modules\@jest\core\build\plugins\FailedTestsInteractive.js
- C:\www\bit-angular\node_modules\.pnpm\registry.npmjs.org+@jest+core@27.3.1_ts-node@9.1.1\node_modules\@jest\core\build\watch.js
- C:\www\bit-angular\node_modules\.pnpm\registry.npmjs.org+@jest+core@27.3.1_ts-node@9.1.1\node_modules\@jest\core\build\cli\index.js
- C:\www\bit-angular\node_modules\.pnpm\registry.npmjs.org+@jest+core@27.3.1_ts-node@9.1.1\node_modules\@jest\core\build\jest.js
- C:\www\bit-angular\node_modules\.pnpm\registry.npmjs.org+jest@27.3.1_ts-node@9.1.1\node_modules\jest\build\jest.js
- C:\www\bit\node_modules\@teambit\jest\dist\jest.tester.js
- C:\www\bit\node_modules\@teambit\jest\dist\jest.main.runtime.js
- C:\www\bit\node_modules\@teambit\bit\dist\load-bit.js
- C:\www\bit\node_modules\@teambit\bit\dist\app.js
- C:\www\bit\bin\bit.js
```

### Additional context

_No response_

### Environment

```shell
System:
    OS: Windows 10 10.0.22000
    CPU: (24) x64 AMD Ryzen 9 5900X 12-Core Processor
  Binaries:
    Node: 16.13.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.4 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 7.19.1 - ~\AppData\Roaming\npm\npm.CMD
```

--- a/package.json
+++ b/package.json
@@ -46,6 +46,7 @@
     "update-e2e": "node scripts/update-e2e.js"
   },
   "dependencies": {
+    "bs-logger": "^0.2.6",
     "esbuild": "0.13.13",
     "jest-environment-jsdom": "^27.0.0",
     "pretty-format": "^27.0.0",

